charts/telepresence/README.md

```diff
@@ -34,6 +34,7 @@ The following tables lists the configurable parameters of the Telepresence chart
 | podCIDRStrategy                                      | Define the strategy that the traffic-manager uses to discover what CIDRs the cluster uses for pods                          | `auto`                                                                      |
 | podSecurityContext                                   | The Kubernetes SecurityContext for the `Pod`                                                                                | `{}`                                                                        |
 | securityContext                                      | The Kubernetes SecurityContext for the `Deployment`                                                                         | `{"readOnlyRootFilesystem": true, "runAsNonRoot": true, "runAsUser": 1000}` |
+| schedulerName                                             | Specify a scheduler for the pod.                                                       |       
```

Comment:
What pod is it that "the pod" here is referencing? Seems the addition applies to multiple pods.